### DISCUSSION TITLE
6bd233ef-79fe-408c-b993-8cf8351460ec

### DISCUSSION THREAD

**Discussion by giscus** [ChooSeoyeon/blog-comment#18] [General]
# 6bd233ef-79fe-408c-b993-8cf8351460ec

소켓(Socket)

https://choo.oopy.io/6bd233ef-79fe-408c-b993-8cf8351460ec

**Comment by andole98** [comment]
HTTP는 단방향적 구조(요청 보내면 응답이 옴)로 통신해서 TCP/IP 프로토콜을 사용하는 소켓처럼 계속 connection이 유지되는 실시간 통신을 할 수 없음

Keep alive header 에 대해 알아보면 좋을것 같아요.
